wordpress-mobile/gutenberg-mobile
# [tech-debt] block-list optimisation

This PR contains some improvements in the performance of block-list

Gutenberg PR - https://github.com/WordPress/gutenberg/pull/22144

To test:
- the app should work the same as before (maybe a bit faster)
- the title should be editable
- inserter point should be rendered in the correct place
- list footer should work - adds new paragraph on click

PR submission checklist:
